org.eclipse.jdt.junit.core/src/org/eclipse/jdt/internal/junit/util/CoreTestSearchEngine.java

```diff
@@ -143,18 +152,40 @@ public static boolean hasJUnit4TestAnnotation(IJavaProject project) {
 	}
 
 	public static boolean hasJUnit5TestAnnotation(IJavaProject project) {
+		return hasJUnitJupiterTestAnnotation(project, 1, // we check JUnit 5 platform bundles, they range in [1.0,2.0)
+				JUnitCore.JUNIT3_CONTAINER_PATH, JUnitCore.JUNIT4_CONTAINER_PATH, JUnitCore.JUNIT6_CONTAINER_PATH);
```

Comment:
`JUNIT6_CONTAINER_PATH` **only**? Not `JUNIT5_CONTAINER_PATH`  && `JUNIT6_CONTAINER_PATH` ?